charts/kms-issuer/templates/rbac.yaml

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
I can't actually get this rbac to generate with `make manifests`, but if we did could be also write it here/into template files?

Reply:
`controller-gen` only create the manager role so not useful here.